app/src/main/java/com/github/fribourgsdp/radio/ImportSpotifyPlaylistsActivity.kt

```diff
@@ -22,6 +24,7 @@ class ImportSpotifyPlaylistsActivity : AppCompatActivity() {
         setContentView(R.layout.activity_import_spotify_playlists)
         val userToken = intent.getStringExtra("auth_token")
         TOKEN = userToken
+        Log.println(Log.INFO, "***", TOKEN.toString())
```

Comment:
Do we need the `print` on the `Log` to stay for `master`?

Reply:
No it is temporary.